components/layout_2020/display_list/stacking_context.rs

```diff
@@ -1059,9 +1061,15 @@ impl BoxFragment {
             );
         }
 
+        let clip_path_chain_id = self.build_clip_path_frame_if_necessary(
+            display_list,
+            containing_block.scroll_node_id,
+            containing_block.clip_chain_id,
+            &containing_block.rect,
+        );
```

Comment:
Why does this happen here rather than near the call to `build_clip_path_frame_if_necessary`. I think that the usage of the terminology `frame` here is left over from when clip where actually "frames" in WebRender. I would omit `frame` in the name of this method.